tests/fsharp/regression/tests_regression.fs

```diff
@@ -0,0 +1,235 @@
+module ``FSharp-Tests-Regression``
+
+open System
+open System.IO
+open NUnit.Framework
+
+open FSharpTestSuiteTypes
+open NUnitConf
+open PlatformHelpers
+
+let testContext = FSharpTestSuite.testContext
+
+
+module ``26`` = 
+
+    [<Test; FSharpSuitePermutations("regression/26")>]
+    let ``26`` p = check (processor {
+        let { Directory = dir; Config = cfg } = testContext ()
+        
+        do! SingleTestBuild.singleTestBuild cfg dir p
+        
+        do! SingleTestRun.singleTestRun cfg dir p
+        })
+
+
+module ``321`` = 
+
+    [<Test; FSharpSuitePermutations("regression/321")>]
+    let ``321`` p = check (processor {
+        let { Directory = dir; Config = cfg } = testContext ()
+        
+        do! SingleTestBuild.singleTestBuild cfg dir p
+        
+        do! SingleTestRun.singleTestRun cfg dir p
+        })
+
+
+module ``655`` = 
+
+    let build cfg dir = processor {
+
+        let exec p = Command.exec dir cfg.EnvironmentVariables { Output = Inherit; Input = None; } p >> checkResult
+        let fsc = Printf.ksprintf (Commands.fsc exec cfg.FSC)
+        let peverify = Commands.peverify exec cfg.PEVERIFY
+
+        // "%FSC%" %fsc_flags% -a -o:pack.dll xlibC.ml
+        do! fsc "%s -a -o:pack.dll" cfg.fsc_flags ["xlibC.ml"]
+
+        // "%PEVERIFY%" pack.dll
+        do! peverify "pack.dll"
+
+        // "%FSC%" %fsc_flags%    -o:test.exe -r:pack.dll main.fs
+        do! fsc "%s    -o:test.exe -r:pack.dll" cfg.fsc_flags ["main.fs"]
+
+        // "%PEVERIFY%" test.exe
+        do! peverify "test.exe"
+
+        }
+
+    let run cfg dir = processor {
+
+        let exec p = Command.exec dir cfg.EnvironmentVariables { Output = Inherit; Input = None; } p >> checkResult
+        let fileguard = (Commands.getfullpath dir) >> FileGuard.create
+
+        // if exist test.ok (del /f /q test.ok)
+        use testOkFile = fileguard "test.ok"
+
+        // %CLIX% test.exe
+        do! exec ("."/"test.exe") ""
+
+        // if NOT EXIST test.ok goto SetError
+        do! testOkFile |> NUnitConf.checkGuardExists
+        }
+
+    [<Test; FSharpSuiteTest("regression/655")>]
+    let ``655`` () = check (processor {
+        let { Directory = dir; Config = cfg } = testContext ()
+
+        do! build cfg dir
+
+        do! run cfg dir
+                
+        })
+
+
+[<Category("fail_new"); Category("fail_reason_ILX_CONFIG")>]
+module ``656`` = 
+
+    let build cfg dir = processor {
+
+        let exec p = Command.exec dir cfg.EnvironmentVariables { Output = Inherit; Input = None; } p >> checkResult
+        let fsc = Printf.ksprintf (Commands.fsc exec cfg.FSC)
+        let peverify = Printf.ksprintf (Commands.peverify exec cfg.PEVERIFY)
+
+        //REVIEW ILX_CONFIG?
+        let ILX_CONFIG = ""
```

Comment:
Assume ILX_CONFIG is always empty - and remove any mention of it on that assumption